dev/PushNotifications/PushNotifications.idl

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Windows.PushNotifications
+{
+    [experimental]
+    // Event args for the Push payload.
+    runtimeclass PushNotificationReceivedEventArgs
+    {
+        // The Push payload
+        byte[] Payload { get; };
+
+        // Gets a deferral to run under specific modes like low power mode
+        Windows.ApplicationModel.Background.BackgroundTaskDeferral GetDeferral();
```

Comment:
It is the second.

Reply:
Can we update the comment to be clearer about what the deferral accomplishes? Usually, the purpose of deferrals is to give the handler time to prepare a response, but here it's so that we can make the system believe that we are still processing the notification in order to take advantage of some hidden undocumented state the system maintains while a notification is in progress.